packages/x-codemod/README.md

```diff
@@ -91,4 +92,37 @@ Renames `locale` into `adapterLocale` (or `LocalizationProvider`)
 npx @mui/x-codemod v6.0.0/localization-provider-rename-locale <path>
 ```
 
+#### `text-props-to-localeText`
+
+Replace props used for translation such as `cancelText` to their corresponding `localeText` key.
+
+```diff
+ <DatePicker
+-  cancelText="Cancel"
++  localeText={{
++    cancelButtonLabel: "Cancel"
++  }}
+ />
+```
+
+```sh
+npx @mui/x-codemod v6.0.0/localization-provider-rename-locale <path>
+```
+
+If you were using always the same text value in all your components, consider moving those translation from the component to the `LocalizationProvider` by hand.
+
+```diff
+<LocalizationProvider
+  dateAdapter={AdapterDayjs}
++ localeText={{ cancelButtonLabel: "Cancel" }}
+>
+ <DatePicker
+-  localeText={{ cancelButtonLabel: "Cancel" }}
+ />
+ <DateTimePicker
+-  localeText={{ cancelButtonLabel: "Cancel" }}
```

Comment:
Maybe having something different than the default makes a bit more sense in this case? 🤔 
```suggestion
+ localeText={{ cancelButtonLabel: "Cancelar" }}
>
 <DatePicker
-  localeText={{ cancelButtonLabel: "Cancelar" }}
 />
 <DateTimePicker
-  localeText={{ cancelButtonLabel: "Cancelar" }}
```